pkg/mutation/logging.go

```diff
@@ -0,0 +1,41 @@
+package mutation
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/google/uuid"
+	"github.com/open-policy-agent/gatekeeper/pkg/logging"
+	"github.com/open-policy-agent/gatekeeper/pkg/mutation/types"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func logAppliedMutations(message string, mutationUUID uuid.UUID, obj *unstructured.Unstructured, allAppliedMutations [][]types.Mutator) {
+	iterations := make([]interface{}, 0, 2*len(allAppliedMutations))
```

Comment:
why 2*len(allAppliedMutations)?

Reply:
There is a key and a value for each entry in `allAppliedMutations`, so the resulting list is twice the length.